mu/package.json

```diff
@@ -7,19 +7,56 @@
     "type": "git",
     "url": "git+https://github.com/mu-editor/Mu-tate.git"
   },
-  "author": "Nicholas H.Tollervey",
+  "author": {
+    "name": "Nicholas H.Tollervey",
+    "email": "ntoll@ntoll.org"
```

Comment:
Hope you don't mind I added your email (taken from the Mu editor configuration), as it is needed to build deb files.